README.md

```diff
@@ -251,7 +251,15 @@ Runs the [`assemble`](#assemble) task when documentation files are changed.
 
 - #### `docs`
 
-This will build a fresh copy of any documentation found in the `docs` directory using Assemble, then call the [`watch`](#watch) task which will watch for any file changes, and finally call the [`browser-sync:docs`](#browser-syncdocs) task which reloads the web page when changes are detected in the `docs/dist` directory.
+Builds a fresh copy of any documentation found in the [`config.docs.rootDir`](#rootdir) directory using Assemble, then watch for any file changes and reload the web page when changes are detected in the [`config.docs.distDir`](#distdir) directory.
```

Comment:
Grammar alert 🚨  Should be '…then **watches** for any file changes and **reloads**…', to match the tense of the previous part of the sentence.